axelrod/tests/integration/test_sample_tournaments.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+import axelrod
+
+C, D = axelrod.Actions.C, axelrod.Actions.D
+
+
+class TestSampleTournaments(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.game = axelrod.Game()
+
+    @classmethod
+    def get_test_outcome(cls, outcome, turns=10):
+
+        # Extract the name of players from the outcome tuples,
+        # and initiate the players by getting the classes from axelrod.
+        names = [out[0] for out in outcome]
+        players = [getattr(axelrod, n)() for n in names]
```

Comment:
Just a thought for the future: we might think about having a dictionary mapping player names to classes in the axelrod module for python beginner friendliness (avoiding the need for `getattr`). Perhaps a solution in search of a problem.